### DISCUSSION TITLE
part-ii/2019-110/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#286] [Question comments]
Part II 2019 Principles of Quantum Mechanics: Paper 3, Section II, B 

https://questions.tripos.org/part-ii/2019-110/

**Comment by thenewtonium** [comment]
<details><summary>Part (a)</summary>

The easiest way to diagonalise $V$ over Span{$\ket{1}, \ket{2}$} is spotting that $\frac{1}{\sqrt{2}}(\ket{1}+\ket{2})$ and $\frac{1}{\sqrt{2}}(\ket{1}-\ket{2})$ are eigenstates with eigenvalues $+\epsilon, -\epsilon$ resp.

</details>

<details><summary>Part (b)</summary>

In this case non-degenerate perturbation theory is fine since the problematic terms $\frac{|\bra{3}V\ket{2}|^2}{E_3 - E_2}$ and $\frac{|\bra{2}V\ket{3}|^2}{E_2 - E_3}$ just vanish anyway.

(Being pernickety: in the derivation of the perturbative expansion, we find the second order correction to the energy to be $E_n^{(2)} = \bra{n}\Delta H\ket{\beta}$ where $\ket{\beta}$ is the coefficient of $\lambda$ in the corresponding energy eigenstate. When $\bra{n}\Delta H\ket{m} = 0$ we actually have $\braket{m|\beta}$ undetermined (rather than $=\frac{\bra{n}\Delta H\ket{m}}{E_m-E_n}$), however the $\ket{m}$ term of $\ket{\beta}$ is just going to vanish when contracted with $\bra{n}\Delta H$...)

</details>
